.circleci/config.yml

```diff
@@ -161,8 +161,6 @@ jobs:
       - run: scripts/clang-tidy-only-diff.sh 4
       - run: make -C build install
       - run: make -C build package
-      - run: make -C build coverage
```

Comment:
There's been discussions about removing coverage checks but we haven't reached a conclusion yet.

Reply:
The hope was rather that it would be possible to speed the process up, rather than removing it entirely.

Reply:
Since non one yet looks at coverage reports, can we make this workflow optional? (not sure if it already is)